lib/shipcloud/operations/all.rb

```diff
@@ -0,0 +1,18 @@
+module Shipcloud
+  module Operations
+    module All
+      module ClassMethods
+        # Loads all Objects of the resource
+        def all
+          response = Shipcloud.request(:get, "#{self.name.split("::").last.downcase}s", {})
+          response.each {|hash| self.new(hash) }
```

Comment:
While this generates the corresponding objects, it still returns the Array of Hashes from the response.
Proposal to amend to get the created objects:

```ruby
response.map { |hash| new(hash) }
```